src/lib/ip-whitelist-backup.json

```diff
@@ -0,0 +1,18 @@
+{
+	"addresses": [
+		"23.235.32.0/20",
```

Comment:
is it wise to store this list in here. Is there potential for the values to be spoofed? config-vars feels like a more natural home, and also allows tying it in to sam's auto key rolling work